src/com/reco1l/framework/ColorComponent.kt

```diff
@@ -52,35 +50,55 @@ data class Color4(private val hex: Long) {
     val blue
         get() = blueInt / 255f
 
-
-    operator fun plus(other: Color4) = Color4(
-        red = red + other.red,
-        green = green + other.green,
-        blue = blue + other.blue,
-        alpha = alpha + other.alpha
-    )
-
-    operator fun minus(other: Color4) = Color4(
-        red = red - other.red,
-        green = green - other.green,
-        blue = blue - other.blue,
-        alpha = alpha - other.alpha
-    )
-
-    operator fun times(other: Color4) = Color4(
-        red = red * other.red,
-        green = green * other.green,
-        blue = blue * other.blue,
-        alpha = alpha * other.alpha
-    )
-
+    /**
+     * Brigthens or darkens the color by multiplying each RGB component by the given
+     * scalar value. Alpha remains unchanged.
+     */
     operator fun times(scalar: Float) = Color4(
         red = red * scalar,
         green = green * scalar,
         blue = blue * scalar,
         alpha = alpha
     )
 
+    /**
+     * Replaces the color alpha with the given scalar value.
+     * Similar to [Tailwind CSS opacity utilities](https://tailwindcss.com/docs/opacity).
+     */
+    operator fun div(scalar: Float) = Color4(
+        red = red,
+        green = green,
+        blue = blue,
+        alpha = scalar
+    )
```

Comment:
I... what? With how `times` work as above, how does this make sense?

I would expect that `color * 2f / 2f` returns the same `color` (in that `color * 2f / 2f == color` evaluates to `true`), but with this, that expectation is thrown off the roof.

Another case: think of how a reader would think if they saw `Color4 / 0f`.

If such method were to exist, I'd use the existing `Color4.copy` and assign the `alpha` directly, which may not look as elegant but is completely within expectations of an outside reader.